packages/cli/src/commands/start.ts

```diff
@@ -22,15 +22,18 @@ import { downloadTemplateAndPlaceInProject, writeEnvKeysToDotEnv } from "../fs";
 import { detectPackageManager, installCommand, runCommand } from "../packages";
 import { execAsync, formatWithPrettier } from "../utils";
 
+export const START_TEMPLATES = [
```

Comment:
this variable is `start templates` ? or `starter template`?

Reply:
OH! maybe starter template is fine, I didn't care actually

Reply:
yeah, it's not important
just i wonder type name and variable name is differnt

Reply:
Oh it's not the same omg I didn't notice